proposals/Exceptions.md

```diff
@@ -1,11 +1,14 @@
 # Exception handling
 
-There are two sections to this proposal
+There are four sections to this proposal
 
 1. A overview of the proposed extension to WebAssembly
+   
+2. Changes to the text format document.
```

Comment:
Here's a cool trick: you can actually number 1. 1. 1. 1. and markdown will autonumber for you. :-)